README.md

```diff
@@ -39,7 +39,7 @@ Google's Urchin Tracking Module & Google Adwords
 - `utm_swu`
 - `gclid`
 
-Adobe Omniture SiteCatalyst
+Adobe Adobe Analytics
```

Comment:
Not that any of these are super important, but why the double-Adobe? Maybe just `Adobe Analytics`?

Reply:
Oh, I am sorry. Will fix it.